File: python/empleado.py
'''
diccionario de empleado
'''

from utilidades import encabezado, linea
from typing import Any, Dict

def main() -> None:
  # Definir un diccionario
  empleado: Dict[str, Any] = {
    "nombre": 'Alfonso',
    'apellido': 'Henriquez',
    'cedula': '34.860.848',
    "edad": 21,
    'sueldo': 1000.00
  }  
  
  encabezado('datos del empleado', 32)
  for key, value in empleado.items():
    print(f'{key.upper()}: {value}')
    
main()

File: Js/src/empleado.js
import { encabezado, line, capitalizar } from "./utilidades.js";

const main = () => {
  const empleado = {
    
  'cedula' : '34.860.848',
  'nombre' : 'Alfonso',
  'apellido' : 'Henriquez',
  'edad' : 18,
  'sueldo' : 1000000.99
  }

encabezado('NOMBRE DEL EMPLEADO', 32)
console.log(`${empleado['nombre']}`)
encabezado('DATOS DEL EMPLEADO', 32)
for (const i in empleado) {
  console.log(`${i.toUpperCase()}: ${empleado[i]}`)
}
}
main()